tests/simulator/refinement/harris.py

```
@@ -0,0 +1,225 @@
+from pyphare.pharein import (
+    ElectromagDiagnostics,
+    FluidDiagnostics,
+    ParticleDiagnostics,
+    InfoDiagnostics,
+)
+from pyphare.pharein import MetaDiagnostics
+from pyphare.pharein import ElectronModel
+from pyphare.simulator.simulator import Simulator, startMPI
+from pyphare.pharein import global_vars as gv
+from pyphare.pharein import LoadBalancer
+import sys
```

Ruff

`sys` imported but unused

Remove unused import: `sys`

(F401)